content/blogposts/2024/cody-vscode-1-10-0-release.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Cody for VS Code v1.10.0 release"
+authors:
+  - name: Kalan Chan
+    url: https://handbook.sourcegraph.com/team/#kalan-chan
+  - name: Justin Dorfman
+    url: https://handbook.sourcegraph.com/team/#justin-dorfman
+publishDate: 2024-03-21T10:00-01:00
+description: "Cody for VS Code v1.10.0 is now available, this release includes support for Claude 3 Haiku, several improvements to doc string generation, and debugging."
+tags: [blog]
+slug: "cody-vscode-1-10-0-release"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0-og-image-2x.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0-og-image-2x.png
+--- 
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v1.10.0 is now available, this release includes support for Claude 3 Haiku, several improvements to doc string generation, and debugging. 
+
+
+## Support links for Cody Pro and Enterprise
+
+We have made it easier for Cody Pro and Enterprise users to get assistance when needed, support links are now more prominently displayed so users can easily access help. Cody Pro and Enterprise users can now get more timely responses by filling out a support form. 
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/support1.png"
+  alt="Support link in the settings panel"
+/>
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/support2.png"
+  alt="Support link in the Cody settings panel"
+/>
+
+## Improved Document Command
+
+Generating documentation from code has been improved to automatically detect documentable code blocks and provide hints to quickly generate doc strings. When your cursor is on a documentable symbol in JavaScript, TypeScript, Go or Python, a "Opt/Alt+D to Document" ghost text will appear. Pressing <kbd>Opt</kbd> / <kbd>Alt</kbd> + <kbd>D</kbd> will immediately generate documentation for that code block.
+
+<Video 
+  source={{
+    mp4: 'blog/vscode-v1-10-Improved-Document-Command-v2'
+  }}
+  loop={true}
+  title="Improved Document Command"
+/>
+
+## GPT-4 Turbo upgrade
+For Cody Pro users, [GPT-4 Turbo](https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo) now uses the `gpt-4-0125-preview` model. 
+
+## Claude 3 Haiku is now available for Cody Pro 
+
+For Cody Pro users we've added Anthropic’s latest AI model, [Claude 3 Haiku](https://www.anthropic.com/news/claude-3-haiku), alongside Sonnet and Opus. This new model offers significantly faster speeds while maintaining quality and coherence. To start using Claude 3 Haiku start a new chat and select it from the list of models:
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/claude3-haiku.png"
+  alt="Claude3 Haiku in the LLM Selection dropdown"
+/>
+
+## New debug options
```

Comment:
Personally think this should be grouped with the `Support links for Cody Pro and Enterprise` entry above, and mention attaching the logs when they file a support ticket can speed things up etc, and then proceed with how they can get the logs with these new Debug command. wdyt?